Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -67,51 +68,78 @@ static String symbolToString(const CSSValue* value)
     return primitiveValue.stringValue();
 }
 
-static CSSCounterStyleDescriptors::Pad translatePadFromStyleProperties(const StyleProperties& properties)
+static CSSCounterStyleDescriptors::AdditiveSymbols translateAdditiveSymbolsFromStyleProperties(const StyleProperties& properties)
 {
-    auto pad = properties.getPropertyCSSValue(CSSPropertyPad).get();
-    if (!pad || !is<CSSValueList>(pad))
-        return CSSCounterStyleDescriptors::Pad();
+    auto additiveSymbols = properties.getPropertyCSSValue(CSSPropertyAdditiveSymbols);
+    if (!additiveSymbols)
+        return { };
+    auto* list = downcast<CSSValueList>(additiveSymbols.get());
+    if (!list)
+        return { };
```

Comment:
There's not much point in a null check here, unless we use `dynamicDowncast`. `downcast` assumes the value is already a CSSValueList.